src/utils/num_utils.ts

```diff
@@ -1,5 +1,12 @@
+import { parseTimeUnit, TimeUnit } from "../constants/time.js";
+
 export const DEFAULT_SIG_FIGS = 5;
 
+const SECONDS_IN_MS = 1000;
+const MINUTES_IN_MS = 1000 * 60;
+const HOURS_IN_MS = 1000 * 60 * 60;
+const DAYS_IN_MS = 1000 * 60 * 60 * 24;
```

Comment:
Minor Nit:
```suggestion
const DAYS_IN_MS = HOURS_IN_MS * 24;
```
and same for other lines above... slightly less error prone